CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Bug fixes
 
 * [#6699](https://github.com/rubocop-hq/rubocop/issues/6699): Fix infinite loop for `Layout/IndentationWidth` and `Layout/IndentationConsistency` when bad modifier indentation before good method definition. ([@koic][])
+* [#6797](https://github.com/rubocop-hq/rubocop/pull/6797): Fix false negative for Layout/SpaceAroundBlockParameters. ([@pocke][])
```

Comment:
Can you explain in more detail about the false negative corrected?

Reply:
Sure :+1: I'll update it.

Reply:
I updated the description of the changelog. Thanks!